webapp/src/js/bootstrapper/purger.js

```diff
@@ -32,10 +32,12 @@ const appendToPurgeList = (localDb, ids) => {
     });
 };
 
-const clearToPurgeList = (localDb) => {
+const removeFromToPurgeList = (localDb, docIds) => {
   return getPurgeLog(localDb)
     .then(log => {
-      log.to_purge = [];
+      log.to_purge = log.to_purge.filter(id => {
+        return docIds.indexOf(id) === -1;
```

Comment:
```suggestion
        return !docIds.includes(id);
```
No magic numbers :D

Reply:
I'm so gun-shy about newfangled features in native code for 3.x. Are you sure that'll work in all browsers?

Reply:
yea, it's added in Chrome 47 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#browser_compatibility

Reply:
Done.